README.md

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
This readme format isn't comprehensive enough, I will create a general template. For now: it's okay